client/asset/eth/fundingcoin.go

```diff
@@ -9,51 +9,61 @@ import (
 	"encoding/binary"
 	"fmt"
 
+	"decred.org/dcrdex/dex"
 	"github.com/ethereum/go-ethereum/common"
 )
 
-const fundingCoinIDSize = 28 // address (20) + amount (8) = 28
+const fundingCoinIDSize = 28      // address (20) + amount (8) = 28
+const tokenFundingCoinIDSize = 36 // address (20) + amount (8) + amount (8) = 36
```

Comment:
Why are there two amounts? Do you mean two addresses, one for the user and one for the token contract? Also, this isn't being used anywhere.

Reply:
Ah makes sense, one is for the gas cost.